ibft/ibft_decided_test.go

```diff
@@ -251,6 +254,52 @@ func TestDecideIsCurrentInstance(t *testing.T) {
 	}
 }
 
+func TestForceDecided(t *testing.T) {
```

Comment:
@nivBlox this doesn't test force decided but rather sync

Reply:
why?